cSpell.json

```diff
@@ -363,6 +363,12 @@
                 "priviledged"
             ]
         },
+        {
+            "filename": "**/specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/WebApps.json",
+            "words": [
+                "sitecontainers"
```

Comment:
Why is this added out of order with regards to api-version?

I noticed a prior entry for Microsoft.Web contained this:

                "Defintions",
                "Availablility",
                "Availabilily",
                "diagnosics"

Do you know why the entry for 2023-12-01 should _only_ contain sitecontainers?

Reply:
![image](https://github.com/Azure/azure-rest-api-specs/assets/74639899/c9d655f2-2070-4679-ac2c-6bbcaf3d3363)

The one you mentioned looks like service.json to me.

sitecontainers are newly added to 2023-12-01.